app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -248,6 +249,9 @@ class EventDetailsFragment : Fragment() {
     }
 
     override fun onPrepareOptionsMenu(menu: Menu?) {
+        val activity = activity as? MainActivity
+        activity?.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        activity?.supportActionBar?.title = ""
```

Comment:
Why here?

Reply:
added as a fail check ,as the ActionBar is changing when going from ticketFragment to Event detail fragment so i invalidated the optionmenu in ticketFragment and set the changing login in onPrepareOptionMenu

Reply:
But it is not an option menu and invalidating option menu should not change action bar.

Reply:
K. I will remove it. but something has to be done, as some issue with the action bar remains in the code , you can see it when going from favoritefragment to eventdetailfragment and back the title becomes events instead of likes I will open a new issue regarding this, as I can see the problem is that the removed fragment is changing the action bar title instead of the one that is going to be displayed.